### PR TITLE
Add restart functionality for ganesha

### PR DESCRIPTION
**Pull request by jschmid1**
Signed-off-by: Joshua Schmid <jschmid@suse.de>

### CODE REVIEW

File: srv/salt/ceph/ganesha/restart/default.sls [hunk 1]
```diff
@@ -0,0 +1,20 @@
+restart:
+  cmd.run:
```
Comment (jan--f) [inline srv/salt/ceph/ganesha/restart/default.sls]: Sorry, I'm out of the loop a little bit...is there a reason for not using the service module? As in service.restart?

Comment (jan--f) [inline srv/salt/ceph/ganesha/restart/default.sls]: hmm nevermind. I guess the answer is as usual "it doesn't do what we need" :(

**Comment by swiftgist** [comment]
I believe the state file can call the service module, but that needs to be verified.  I suggest we create a separate issue if that's fine with each of you.

**Comment by jan--f** [comment]
@swiftgist absolutely

**Comment by jschmid1** [comment]
@swiftgist fine for me

**Comment by swiftgist** [comment]
Created #178

**Pull request merged**